### PR TITLE
Static page and using Helper

### PR DESCRIPTION
**Pull request by anhkhang2304**
Chapter 3 combined with Chapter 4

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,14 +1,13 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
+    <title><%= full_title(yield(:title)) %></title>
     <%= csrf_meta_tags %>
     <%= stylesheet_link_tag    'application', media: 'all',
```
Comment (thangtx88) [inline app/views/layouts/application.html.erb]: <%= stylesheet_link_tag    "application", media: "all",
                                                "data-turbolinks-track": "reload" %>

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title(page_title = '')
```
Comment (HoangThiNhung) [inline app/helpers/application_helper.rb]: remove ()

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (thangtx88) [inline app/views/layouts/application.html.erb]: use double quote

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -1,14 +1,13 @@
...
                                               'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (thangtx88) [inline app/views/layouts/application.html.erb]: use double quote

File: config/routes.rb [hunk 4]
```diff
@@ -4,6 +4,5 @@
   get 'static_pages/home'
```
Comment (thangtx88) [inline config/routes.rb]: use double quote

File: config/routes.rb [hunk 5]
```diff
@@ -4,6 +4,5 @@
...
   get  'static_pages/about'
```
Comment (thangtx88) [inline config/routes.rb]: use double quote

File: config/routes.rb [hunk 6]
```diff
@@ -4,6 +4,5 @@
...
   get 'static_pages/help'
```
Comment (thangtx88) [inline config/routes.rb]: use double quote

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (thangtx88) [inline app/helpers/application_helper.rb]: use double quote

Comment (thangtx88) [inline app/helpers/application_helper.rb]: remove ()

File: app/helpers/application_helper.rb [hunk 7]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ''
```
Comment (HoangThiNhung) [inline app/helpers/application_helper.rb]: double quote

File: app/views/layouts/application.html.erb [hunk 8]
```diff
@@ -1,14 +1,13 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
+    <title><%= full_title(yield(:title)) %></title>
```
Comment (HoangThiNhung) [inline app/views/layouts/application.html.erb]: remove ()

**Pull request merged**